torchvision/models/inception.py

```diff
@@ -24,47 +24,16 @@
 _InceptionOutputs = InceptionOutputs
 
 
-def inception_v3(pretrained=False, progress=True, **kwargs):
```

Comment:
Although it seems right to move this below the definition of `Inception3` we could also annotate it with quotes, i.e. `-> "Inception3"`. The latter while worse from a design stand point would reduce the amount of changes. @fmassa?

Reply:
I would prefer fewer code-movements, so let's annotate with `"Inception3"` instead.

Reply:
Done!